src/App.js

```diff
@@ -304,9 +325,14 @@ class App extends Component {
   toMainView = () => {
     this.setState({ 'currentView': View.DASHBOARD });
   }
-  
+
   getLayoutsList = () => {
-    return (Object.keys(this.store.ls).filter(function (propertyName) { return propertyName.indexOf("layout") === 0;}));
+    const keys = this.store.getKeys();
+    if (keys) {
```

Comment:
I think if you return [] instead of null in store.getKeys when this.ls is undefined, then you could safely get rid of this test here and the return [] at line 334.